src/1_Introduction/1_Intro.ipynb

```diff
@@ -219,29 +219,29 @@
+    "Amorphous materials do not have long-range lattice regularities. However, covalent compounds can induce short-range regularities in the lattice structure, such as fixed coordination number $z$. The best examples are amorphous Silicon and Germanium with $z=4$ which are used to make thin-film solar cells\\ [@Weaire1971; @betteridge1973possible]. Recently, it has been shown that topological insulating phases can exist in amorphous systems. Amorphous topological insulators are characterized by similar protected edge states to their translation invariant cousins and generalised topological bulk invariants\\ [@mitchellAmorphousTopologicalInsulators2018; @agarwala2019topological; @marsalTopologicalWeaireThorpeModels2020; @costa2019toward; @agarwala2020higher; @spring2021amorphous; @corbae2019evidence]. However, research on amorphous electronic systems has mostly focused on non-interacting systems with a few exceptions, for example, to account for the observation of superconductivity\\ [@buckel1954einfluss; @mcmillan1981electron; @meisel1981eliashberg; @bergmann1976amorphous; @mannaNoncrystallineTopologicalSuperconductors2022] in amorphous materials or very recently to understand the effect of strong electron repulsion in TIs\\ [@kim2022fractionalization].     \n",
     "\n",
-    "Amorphous _magnetic_ systems have been investigated since the 1960s, mostly through the adaptation of theoretical tools developed for disordered systems\\ [@aharony1975critical; @Petrakovski1981; @kaneyoshi1992introduction; @Kaneyoshi2018] and with numerical methods\\ [@fahnle1984monte; @plascak2000ising]. Research on classical Heisenberg and Ising models has been shown to account for observed behaviour of ferromagnetism, disordered antiferromagnetism and widely observed spin glass behaviour\\ [@coey1978amorphous]. However, the role of spin-anisotropic interactions and quantum effects in amorphous magnets has not been addressed. \n",
+    "Amorphous _magnetic_ systems have been investigated since the 1960s, mostly through the adaptation of theoretical tools developed for disordered systems\\ [@aharony1975critical; @Petrakovski1981; @kaneyoshi1992introduction; @Kaneyoshi2018] and with numerical methods\\ [@fahnle1984monte; @plascak2000ising]. Research on classical Heisenberg and Ising models accounts for observed behaviour of ferromagnetism, disordered antiferromagnetism and widely observed spin glass behaviour\\ [@coey1978amorphous]. However, the role of spin-anisotropic interactions and quantum effects in amorphous magnets has not been addressed. \n",
```

Comment:
```suggestion
    "Amorphous _magnetic_ systems have been investigated since the 1960s, mostly through the adaptation of theoretical tools developed for disordered systems\\ [@aharony1975critical; @Petrakovski1981; @kaneyoshi1992introduction; @Kaneyoshi2018] and with numerical methods\\ [@fahnle1984monte; @plascak2000ising]. Research on classical Heisenberg and Ising models accounts for the observed behaviour of ferromagnetism, disordered antiferromagnetism and widely observed spin glass behaviour\\ [@coey1978amorphous]. However, the role of spin-anisotropic interactions and quantum effects in amorphous magnets has not been addressed. \n",
```